styles.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-size: 18px;
+}
+
+main {
+  width: 100%;
+  height: 100%;
+}
+
+.container {
+  width: 100%;
+  max-width: 450px;
+}
+
+header {
+  background-color: #3c3a39;
+  width: 100%;
+  height: 5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 1rem;
+}
+
+.logo-container {
+  width: fit-content;
+}
+
+.logo-container p {
+  color: white;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 700;
+  font-family: 'Roboto', serif;
+}
+
+.home {
```

Comment:
Kindly make sure CSS Flexbox to position your sections and elements as stated in the [Project Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox%20in%20all%202%20sections.) :eyes: :memo: